src/Docker/OutputFilter/OutputFilter.php

```diff
@@ -2,14 +2,21 @@
 
 namespace Keboola\DockerBundle\Docker\OutputFilter;
 
+use Keboola\DockerBundle\Docker\Container\WtfWarningFilter;
+use function Keboola\Utils\sanitizeUtf8;
+
 class OutputFilter implements OutputFilterInterface
 {
-    const REPLACEMENT = '[hidden]';
+    private const REPLACEMENT = '[hidden]';
+    private const TRIMMED = '[trimmed]';
 
-    /**
-     * @var array
-     */
-    private $filterValues = [];
+    private array $filterValues = [];
+    private int $maxMessageChars;
+
+    public function __construct(int $maxMessageChars)
```

Comment:
Do we not want to use a default here?

Reply:
I couldn't come up with a logical default 😆

Reply:
hmm, looks like you liked 10000 and 10^6 in the tests 😄 .  Yeah, I don't know what it should be either 😄